### PR TITLE
Feat: Adds tokens to `Price`

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `Price`: new scoped tokens linked to global ones.

## How does it work?

This PR uses local variables to parameterize the `Price` properties, then connects these scoped tokens to the global ones.

#### Listing

| Local token                                          | Default value/Global token linked                                                                        |
|-|-|
| <code>--fs-price-listing-color</code>                | <code>var(--fs-color-text-light)</code> |
| <code>--fs-price-listing-text-decoration-line</code> | <code>line-through</code> |

#### Spot

| Local token                              | Default value/Global token linked       |
|-|-|
| <code>--fs-price-spot-margin-left</code> | <code>var(--fs-spacing-1)</code>        |
| <code>--fs-price-spot-font-weight</code> | <code>var(--fs-text-weight-bold)</code> |

## How to test it?

The `Price` component and its variants should look just as it was before these changes.

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

- PR description
- [x] Updated the Storybook - *if applicable*.
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.
- [x] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

**Review by hellofanny — APPROVED** [review]
After merging https://github.com/vtex-sites/gatsby.store/pull/83 I think this one is ready to merge too! 🚀

File: src/components/ui/Price/Price.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+| Local token                                     | Default value/Global token linked                                                                        |
+| ----------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| <code>--fs-price-listing-color</code>           | <div style={{ backgroundColor: 'var(--fs-color-text-light)' }} /><code>var(--fs-color-text-light)</code> |
+| <code>--fs-price-listing-text-decoration</code> | <code>line-through</code>                                                                                |
```
Comment (renatamottam) [inline src/components/ui/Price/Price.stories.mdx:141]: Could you change it to the new `TokenTable` component?

**Review by renatamottam — APPROVED** [review]
Approved with a request 😬

**Review by lucasfp13 — APPROVED** [review]
LGTM!

File: src/components/ui/Price/Price.stories.mdx (hunk 1 above)
Comment (eduardoformiga) [reply src/components/ui/Price/Price.stories.mdx]: Sure. Done [here](https://github.com/vtex-sites/nextjs.store/pull/80/commits/0ece4c1f07f9e716cb67ba6cdf69912fa2292759).

**Pull request merged**